src/annotator/anchoring/pdf.js

```diff
@@ -377,19 +406,23 @@ function prioritizePages(position) {
     return result;
   }
 
-  return findPage(position.start).then(({ index }) => sortPages(index));
+  return findPage(position).then(({ index }) => sortPages(index));
 }
 
 /**
  * Anchor a set of selectors to a DOM Range.
```

Comment:
I know this is inherited, but what does "Anchor" even mean? This is partly a rhetorical, partly a serious question.

Reply:
I guess it's a convenient piece of project/domain-specific jargon we've ended up with. We could use "resolve" or "evaluate" I guess, although "anchor" is more specific in context.